src/test/scala/viper/gobra/GobraTests.scala

```diff
@@ -29,12 +30,17 @@ class GobraTests extends AbstractGobraTests with BeforeAndAfterAll {
   override val defaultTestPattern: String = PackageResolver.inputFilePattern
 
   var gobraInstance: Gobra = _
+  var executor: GobraExecutionContext = _
 
   override def beforeAll(): Unit = {
+    executor = new DefaultGobraExecutionContext()
     gobraInstance = new Gobra()
   }
 
   override def afterAll(): Unit = {
+    executor.terminateAndAssertInexistanceOfTimeout()
+    ViperServerBackend.resetServer();
+    ViperServerBackend.resetExecutor();
```

Comment:
why do we do this here?

Reply:
I actually updated `ViperBackends` such that we don't need this here